src/reuser/agent/Makefile

```diff
@@ -13,7 +13,7 @@ include $(VARS)
 COPY = reuser.php version.php ReuserAgent.php
 WRAP = reuser
 
-all: version.php
+all: version.php reuser
```

Comment:
It will be better to add the `reuser` dependency in test recipe instead in `all` as it is only meant for testing (and is throwing [some warnings](https://travis-ci.org/fossology/fossology/jobs/436563120#L3446) as well)